src/client/src/orangehrmRecruitmentPlugin/pages/OfferDeclinedAction.vue

```diff
@@ -0,0 +1,96 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <oxd-form :loading="isLoading">
+    <recruitment-status :data="data">
+      <template #header-title>
+        <oxd-text tag="h6" class="orangehrm-main-title">
+          {{ $t('recruitment.application_stage') }}
+        </oxd-text>
+      </template>
+      <template #footer-title>
+        <oxd-text type="subtitle-2"> Status: {{ action.label }} </oxd-text>
+      </template>
+      <template #footer-options>
+        <oxd-button
+          display-type="ghost-danger"
+          :label="$t('recruitment.reject')"
+          @click="onReject"
+        />
+      </template>
+    </recruitment-status>
+  </oxd-form>
```

Comment:
why can't we keep one component and only change button? I feel this component is duplicated needlessly 

```
<oxd-form :loading="isLoading">
    <recruitment-status :data="data">
      <template #header-title>
        <oxd-text tag="h6" class="orangehrm-main-title">
          {{ $t('recruitment.application_stage') }}
        </oxd-text>
      </template>
      <template #footer-title>
        <oxd-text type="subtitle-2"> Status: {{ action.label }} </oxd-text>
      </template>
      <template #footer-options>
        <oxd-button
          display-type="ghost-danger"
          :label="$t('recruitment.reject')"
          @click="onReject"
        />
      </template>
    </recruitment-status>
  </oxd-form>
```